website/meta/universe.json

```diff
@@ -1,5 +1,27 @@
 {
     "resources": [
+	{
+	    "id": "numerizer",
+	    "title": "numerizer",
+	    "slogan": "Convert natural language numerics into ints and floats.",
+	    "description": "A SpaCy extension for Docs, Spans and Tokens that converts numerical words and quantitative named entities into numeric strings.",
+	    "github": "jaidevd/numerizer",
+	    "pip": "numerizer",
+	    "code_example": [
+		"from spacy import load",
+		"import numerizer",
+		"nlp = load('en_core_web_sm') # or any other model",
+		"doc = nlp('The Hogwarts Express is at platform nine and three quarters')",
+		"doc._.numerize()",
```

Comment:
I haven't looked in the code in detail, but I'm not sure I understand the API here. The `numerize()` method is called on spaCy's `Underscore` method? Why not add a [custom pipeline](https://spacy.io/usage/processing-pipelines#custom-components) to `nlp` instead that sets a [custom attribute](https://spacy.io/usage/processing-pipelines#custom-components-attributes)?

Reply:
I found calling a `.set_extension` on the `Doc` - and specifying it's methods / getters / setters a lot easier than adding a custom pipeline component. Additionally, it seems like adding a custom pipeline component set's an extra step for the user (calling `.add_pipe` before using the pipeline.) I thought that this way, all one has to do is `import numerizer` and their extension is ready.

Is this a valid assessment? If not, I can look into the custom component.

Reply:
Ultimately the choice is up to you, ofcourse, as you're designing the plugin. I just felt like many spaCy users would be looking to add a custom component from your library, as it seems like the "standard approach". I was actually confused myself for a second that that's not happening. But like I said, the choice of UI is yours, just wanted to share my 2 cents ;-)

Reply:
I'll merge this as-is for now, in case you'd ever want to change the API on your side, you can always send a new PR with an update!